src/content/docs/tutorial-errors/solve-critical-errors.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Solve your critical errors
+metaDescription: Learn how to manage a high number of errors in your app.   
+---
+
+import apmErrorsAttributes from 'images/apm_screenshot-crop_errors-details.webp'
+
+import apmLogs from 'images/apm_screenshot-crop_errors-logs.webp'
+
+import apmLogsZoom from 'images/apm_screenshot-crop_errors-logs-zoomed.webp'
+
+import apmStackTraces from 'images/apm_screenshot-crop_errors-stack-traces.webp'
+
+import apmErrorIDE from 'images/apm_screenshot-crop_errors-IDE.webp'
+
+Once you've chosen your error group, New Relic takes you to a new page that displays attributes, logs, and traces about the error. This part of the tutorial guides you through two recommended avenues for analyzing an error, your logs and stack traces. 
+
+## Objectives [#objectives]
+
+This doc builds off the problem scoped from the [Respond to service outages](/docs/tutorial-errors/respond-outages) doc of this tutorial series. In this doc, you will:
```

Comment:
Should respond be capitalized?

Reply:
I think so because it's referring to a title in the doc. This is how I usually stylize doc titles when they're inside a sentence